postgres/pgxv4/postgres.go

```diff
@@ -59,39 +59,38 @@ type pgDriver struct {
 //
 // "host=my-project:us-central1:my-db-instance user=myuser password=mypass"
 func (p *pgDriver) Open(name string) (driver.Conn, error) {
-	var (
-		dbURI string
-		ok    bool
-	)
+	dbURI, err := p.dbURI(name)
+	if err != nil {
+		return nil, err
+	}
+	return stdlib.GetDefaultDriver().Open(dbURI)
 
-	p.mu.RLock()
-	dbURI, ok = p.dbURIs[name]
-	p.mu.RUnlock()
+}
 
+func (p *pgDriver) dbURI(name string) (string, error) {
```

Comment:
should we adding a comment above this func explaining it? Even if its only a one-line comment